lemon/swagger.yaml

```diff
@@ -189,6 +189,75 @@ paths:
               schema:
                 $ref: "#/components/schemas/GetProductStructureResult"
 
+  /api/production/machines:
+    get:
+      tags:
+        - Machine
+      summary: Get a list of machines
+      operationId: listMachines
+      parameters:
+        - name: filter.code
+          in: query
+          schema:
+            type: string
+        - name: filter.description
+          in: query
+          schema:
+            type: string
+        - name: filter.type
+          in: query
+          schema:
+            type: integer
+            format: int32
+        - name: filter.hour_price
+          in: query
+          schema:
+            type: number
+            format: double
+        - name: filter.capacity
+          in: query
+          schema:
+            type: number
+            format: double
```

Comment:
Probably not needed

Reply:
isn't it still better to include all the parameters from official docs just in case?